recipes/libgpg-error/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+./configure --prefix=$PREFIX
+
+make
+make install
```

Comment:
Not without increasing the build requirements and complexity (gtest, gmock), which I couldn't actually get working.